frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -104,9 +104,15 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
     }
   }
 
+  const accessReview = createAccessReview || {
+    model: k8sModel,
+    namespace,
+    groupVersionKind,
+  };
```

Comment:
We shouldn't need both `model` and `groupVersionKind`. It should be one or the other. Can you check which is right and ideally specify a type for `accessReview` to benefit from type checking?

Reply:
yes, model is not necessary. the type is
```
createAccessReview?: {
    groupVersionKind: GroupVersionKind;
    namespace?: string;
  };
```